zhaquirks/xiaomi/aqara/switch_acn047.py

```diff
@@ -206,5 +207,12 @@ class AqaraT2Relay(XiaomiCustomDevice):
     }
 
     device_automation_triggers = {
-        (SHORT_PRESS, SHORT_PRESS): {COMMAND: SINGLE},
+        (SHORT_PRESS, PRESS_TYPES.get(1)): {
```

Comment:
Use `PRESS_TYPES[1]` instead of `.get(1)` here, as we know the values exist (or should exist at least).